packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -12,55 +12,95 @@ import {useIntl} from 'react-intl'
 import {useCurrency} from '@salesforce/retail-react-app/app/hooks'
 
 const DisplayPrice = ({
-    basePrice,
-    discountPrice,
+    strikethroughPrice,
```

Comment:
As per suggestion on #1754 , I've refactored this component to have `currentPrice` and `strikethroughPrice` props instead of `basePrice` and `discountPrice` to make it less confusing and more clarity

Reply:
I think we are part way there. The issue Joel and myself was trying to get across was that in the jsx both the list and strikethough prices are conditionally rendered. So reading it at a glance it looks a little weird. 

But I think this really stems from the implementation detail that we chose to use a Skeleton internally in this component. That decision made the implementation of this component a lot more complex then is has to be. But using the skeleton internally we:

1. essentially make every prop optional, if we didn't have the skeleton, then we could say that the basePrice is required and we wouldn't have to conditional render it and it would be easier to render. 
2. Also we wouldn't have to do that condition for the translations either.

So I would recommend that we take the skeleton out (yes we would have to use a skeleton in multiple other places, but don't know if having a skeleton on this granular of a level is the right approach. Also, I would remove the `isProductSet` prop and replace it with something more generic, like `labelPrefix`, that way we it adds more flexibility to the component in how it can be used.

Reply:
Thanks for the feedback. I've pushed the changes as per suggestion.
- Removed skeleton
- currentPrice is a mandatory prop
- strikethroughPrice can be render conditionally